package.json

```diff
@@ -11,21 +11,23 @@
   },
   "dependencies": {
     "@ar.io/sdk": "^3.9.1",
+    "@ardrive/turbo-sdk": "^1.23.4-alpha.1",
     "@arweave-wallet-kit/browser-wallet-strategy": "^0.1.1",
     "@arweave-wallet-kit/core": "^0.1.1",
     "@arweave-wallet-kit/othent-strategy": "^0.1.1",
     "@arweave-wallet-kit/react": "^0.3.0",
     "@arweave-wallet-kit/styles": "^0.1.1",
     "@arweave-wallet-kit/wander-strategy": "^0.1.2",
     "@arweave-wallet-kit/webwallet-strategy": "0.0.1-beta.4",
+    "arweave-wallet-connector": "^1.0.2",
+    "bitcoinjs-lib": "^6.1.7",
```

Comment:
we could roll this into turbo-sdk, it is a `peerDependency` of `@keplr-wallet/crypto`